internal/core/textlayout/sharedparley.rs

```diff
@@ -1213,6 +1221,66 @@ pub fn draw_text(
     }
 }
 
+#[cfg(feature = "experimental-rich-text")]
+pub fn handle_links(
```

Comment:
Perhaps this function could be called link_under_cursor()? Handle looks a little a abstract when this function is concrete by returning a found link :)